Readme.md

```diff
@@ -12,6 +12,8 @@ This is a front-end for the wonderful [**Fantomas**](https://github.com/fsprojec
 - Fully asynchronous, does not increase project load times
 - Loaded when required
 - Respects key-bindings
+- Not coupled to a specific version of Fantomas. Works with whichever version is installed on your machine
```

Comment:
Suggest to clarify that it works with whatever is installed in your repo? (which I think it does?)

Reply:
I know the next line sort of clarifies but "installed on your machine" can be a bit misleading?

Reply:
Maybe mention it uses `Fantomas.Client` and link to [this doc](https://fsprojects.github.io/fantomas/docs/end-users/VSCode.html#Fantomas-version-detection-used-by-Fantomas-Client).